substrate/frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -510,6 +510,17 @@ impl<T: Config> Pallet<T> {
 		let positive_imbalance =
 			T::Currency::deposit_creating(who, reward.saturating_add(call_fee));
 		T::RewardHandler::on_unbalanced(positive_imbalance);
+
+		// `who` is force inserted at the top of the whitelist if it is not whitelisted yet,
+		// potentially pushing out older whitelisted accounts. If `who` is already part of the
+		// whitelist, move it to the head of the whitelist, while keeping the order of the remaining
+		// elements.
+		SignedWhitelist::<T>::mutate(|w| {
```

Comment:
alternative: 
- use `BoundedVecDeq` as it has `push_front`. 
- always start with removing the existing one. 
- then pushing to front